src/core/scene/a-scene.js

```diff
@@ -507,6 +510,80 @@ module.exports.AScene = registerElement('a-scene', {
       writable: true
     },
 
+    setupLoader: {
```

Comment:
Can place all the loader code in a module that exports functions `setupLoader(sceneEl)` to not pollute scope, not meant to be callable by user